tools/demo-gui/demo-gui/sof_demo_gui.py

```diff
@@ -63,7 +63,7 @@ def apply_css(self):
             color: #FFFFFF;
```

Comment:
@Prjbond007 update the commit message to include your Signed-off-by - see https://docs.kernel.org/process/submitting-patches.html

tools: demo-gui: fix CSS styling headerbar.titlebar

Updated the headerbar.titlebar in apply_css function to background-color for consistency.

Signed-off-by: Your_name <your_email_address>